datafusion/core/tests/sqllogictests/test_files/aggregate.slt

```diff
@@ -1390,6 +1429,63 @@ as values
  ('2021-01-01T05:11:10.432', 'Row 3');
 
 
+statement ok
```

Comment:
Given postgres also runs these queries what do you think about also adding them into a `pg_compat` test in https://github.com/apache/arrow-datafusion/tree/main/datafusion/core/tests/sqllogictests/test_files/pg_compat?